main.py

```diff
@@ -680,6 +680,7 @@ def handle_switch_maintenance_start(self, event):
 
     def handle_link_down(self, interface):
         """Notify a link is down."""
+        interface.deactivate()
```

Comment:
As we discussed, by deactivating the interface here, the link will automatically become `active=False` (as per https://github.com/kytos-ng/kytos/blob/master/kytos/core/link.py#L60-L61) and no event `kytos/topology.link_down` will be generated (which impacts mef_eline and others). Maybe we can move the `interface.deactivate()` down to line 689 (after `notify_link_status_change()`, outside of the `if`)

Reply:
done https://github.com/kytos-ng/topology/pull/37/commits/9d1a54f83f077a9a0f17266802baf517a9863e4e

Reply:
Checked and it looks good to me!